bin/Cargo.toml

```diff
@@ -55,6 +56,7 @@ ethui-token-list.workspace = true
 
 [features]
 default = []
+forge-traces = ["ethui-args/forge-traces", "ethui-forge-traces/forge-traces", "ethui-rpc/forge-traces"]
```

Comment:
why does the forge-traces crate have a forge-traces feature flag?
you can just mark that crate as `optional = true`, and add `dep:ethui-forge-traces` here to include it as dependency when the feature is enabled

the crate itself doesn't need to know about the feature flag